Dockerfile

```diff
@@ -4,15 +4,16 @@ ARG LIBSIG_VERSION=3.0.3
 ARG CARES_VERSION=1.24.0
 ARG CURL_VERSION=8.5.0
 ARG XMLRPC_VERSION=01.58.00
-ARG LIBTORRENT_VERSION=v0.13.8
-ARG RTORRENT_VERSION=v0.9.8
 ARG MKTORRENT_VERSION=v1.1
 ARG GEOIP2_PHPEXT_VERSION=1.3.1
 
 # v4.3.1
 ARG RUTORRENT_VERSION=97b431479022ceb1cc81df6b7e234e606bd3e13c
 ARG GEOIP2_RUTORRENT_VERSION=4ff2bde530bb8eef13af84e4413cedea97eda148
 
+# rTorrent stickz v3
+ARG RTORRENT_STICKZ_VERSION=491dc6ed1ab3744d3b14800c33550564204bc8f5
```

Comment:
I see https://github.com/stickz/rtorrent embeds both rtorrent and libtorrent. So I guess we need to change the versioning strategy here to better track changes.

Currently we have the versioning based on rTorrent, libtorrent and ruTorrent version and also associated patches:

`<rutorrent_ver>-<rtorrent_ver>_<rtorrent_buildnum>-<libtorrent_ver>_<libtorrent_buildnum>-<release>`

If we want to correctly track changes in upcoming releases of your repo, I think you should adopt something similar on your side with your Git tags. For example [`v3`](https://github.com/stickz/rtorrent/releases/tag/v3) does not explicitly tells the rtorrent and libtorrent version. I think you should include build metadata (rtorrent and libtorrent versions) associated and probably adopt a proper [semver](https://semver.org/) like `v3.0.0+0.9.8-0.13.8`.

Then on my side I would just tag `<ruTorrent_ver>-<rtorrent-stickz_ver>-<release>`. For example: `4.3.1-3.0.0-0`.

WDYT?

Reply:
Sounds good. I have retagged the last couple releases. For instance [v3.0-0.9.8-0.13.8](https://github.com/stickz/rtorrent/tree/v3.0-0.9.8-0.13.8).

1. No patch releases for this repo. Just major + minor.
2. Need to maintain the `-` to avoid URL symbols.